tests/integration/testdata/sync/infra/cdk/cdk_v1_synthesized_template_level3_nested_zip_functions.json

```diff
@@ -52,7 +52,7 @@
         "aws:asset:path": "before/asset.b998895901bf33127f2c9dce715854f8b35aa73fb7eb5245ba9721580bbe6948",
         "aws:asset:is-bundled": false,
         "aws:asset:property": "Content",
-        "BuildMethod": "python3.8"
```

Comment:
Why this python version change required?

Reply:
windows does not have python3.8